stripe/src/main/java/com/stripe/android/PaymentConfiguration.java

```diff
@@ -61,6 +62,15 @@ public PaymentConfiguration setShouldUseSourcesForCards(boolean shouldUseSources
         return this;
     }
 
+    void setEphemeralKeyProviderClassLoader(@NonNull ClassLoader classLoader) {
```

Comment:
We would have to make CustomerSession a singleton to do that in order to keep the ClassLoader _somewhere_ if we don't just store its ClassLoader in our only singleton thus far. But if `CustomerSession` is a singleton, then I can shave off about half of this PR because we don't need to bother with Parcelable